rfp-proposals/rfp-proposal-filemeta.io.md

```diff
@@ -0,0 +1,147 @@
+# RFP Proposal: `filemeta.io`
+
+**Name of Project: filemeta.io**
+
+**Link to RFP:[rfp-block-explorer](/rfps/rfp-block-explorer.md)**
+
+**RFP Category:app-dev**
+
+**Proposer:[@strahe](https://github.com/strahe)**
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:Yes**
+
+# Project Description
+
+We call the project'filemeta.io'. As most blockchain explorers,'filemeta.io' can display and search most detailed blockchain metrics of Filecoin blockchain,such as Block Data, Deals, Actors,etc.
+
+The performance of current blockchain explorers shows that traditional data storage solution has been adopted extensively in most long run projects with good experience,like `mongodb sharding cluster`, `elasticsearch`.We also agree that there is a big advantage in big data query for using traditional data storage solution rather than standalone HTTP API. Based on this idea, we'll focus on providing more statistical analysis data,better query performance and higher availablity in addition to basical data display and query function of explorers.
+
+In general,`filemeta.io` has advantages of block data storage solution optimized for big data ,more statistical analysis data and higher availablity.
+
+## Deliverables
+
+Final deliverables will include:
+
+* Fully functional block explorer website(Build with `Python3 asyncio` and `Vuejs`).
+  * Publicly accessible via `https://`.
+  * `Mobile-responsive` and `Real-time` design(Websocks first but compatible with polling mode).
+  * Fast to load.
+  * Good UE && UI(Will provide a High-fidelity design mockups).
+  * Highly available.
+* Open-sourced repo for all work on this project.
+  * Well-documented.
+  * Dual-licensed under MIT and APACHE.
+* Backend API documentation(With swagger UI).
+* A generic API for querying backend data(Maybe it can be used as a supplement to the Filecoin REST API?).
+
+## Development Roadmap
+
+Milestones:
+
+| No. | Description | Funding | Estimated Timeframe |
+| ----------- | ----------- | ----------- | ----------- |
+| 1   | Visual design and animation design.    | $5000 | 3 weeks |
```

Comment:
What animation design do you anticipate needing for this project

Reply:
We prefer eospark.com, but bloks.io is also good. (Depending on EOSIO and Filecoin, some features may be removed.)

Reply:
Are you referring to the visual design and graphics of those websites? I don't see any animations per se.

Reply:
Yes, it's just visual design and graphics. We haven't really started designing yet, but our goals are similar to their designs.

Reply:
Sorry, the above mentioned is bloks.io not blocks.io